infrastructure/output.tf

```diff
@@ -8,26 +8,18 @@ output "vaultName" {
   value = "${module.local_key_vault.key_vault_name}"
 }
 
-output "idam_api_url" {
+output "idam_api_base_uri" {
   value = "${var.idam_api_url}"
```

Comment:
worth changing the variables.tf name to idam_api_base_url too?

Reply:
yes